### ISSUE TITLE
[Feature] Persistent volume for incognito browsing

### ISSUE THREAD

**Issue by BenKan** [stashapp/stash#5162]
If I set the volume while not incognito browsing, stashapp recalls my volume e.g. 50%. Contrary to that, using an incognito browser, the volume is not saved and is always at 100%.

It would be great, if the volume setting is remembered even for incognito browsing.

**Comment by Gykes** [comment]
Correct me if I'm wrong here. 

When you go into incognito all settings/cookies are lost/forgotten? If I'm not mistaken that's the point of incognito. It wouldn't be possible to make the sound level stick without altering the point of incognito, right?

**Comment by echo6ix** [comment]
> Correct me if I'm wrong here.
> 
> When you go into incognito all settings/cookies are lost/forgotten? If I'm not mistaken that's the point of incognito. It wouldn't be possible to make the sound level stick without altering the point of incognito, right?

You're correct.

He's basically asking for persistent volume across devices/browsers, which I think is a bad idea. Users lose to the ability to vary volume per device/browser, resulting in an inconsistent user experience. This is an obvious deficiency so maybe he just wants to implement a toggle setting to enable this specific use case. I think this would be better suited for a plugin FR.

**Comment by Gykes** [comment]
> > Correct me if I'm wrong here.
> > When you go into incognito all settings/cookies are lost/forgotten? If I'm not mistaken that's the point of incognito. It wouldn't be possible to make the sound level stick without altering the point of incognito, right?
> 
> You're correct.
> 
> He's basically asking for persistent volume across devices/browsers, which I think is a bad idea. Users lose to the ability to vary volume per device/browser, resulting in an inconsistent user experience. This is an obvious deficiency so maybe he just wants to implement a toggle setting to enable this specific use case. I think this would be better suited for a plugin FR.

Agreed, most users want anonymity and I feel like this could potentially pull from that. My vote is Stash shouldn't be changing how incognito works. Plugin idea from me as well.
